<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexWidget</class>
 <widget class="QWidget" name="IndexWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: &quot;#151515&quot;;</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>670</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/page_index.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QQuickWidget" name="map_webview">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius: 15px;border-bottom-right-radius: 15px; background-color: #151515;</string>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QQuickWidget" name="car_webview">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoMusic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSOS">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: red; color: white; font-size: 18px; border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>SOS</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
